src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessible.java

```diff
@@ -184,6 +184,11 @@ public void propertyChange(PropertyChangeEvent e) {
                     if (thisRole == AccessibleRole.CHECK_BOX) {
                         valueChanged(ptr);
                     }
+
+                    // Do send radio button state changes to native side
+                    if (thisRole == AccessibleRole.RADIO_BUTTON) {
+                        valueChanged(ptr);
+                    }
```

Comment:
This looks fine. do we need to implement the same logic for the togglebutton?

Reply:
As you mentioned, I checked for togglebutton and it requires the same logic implementation.
Added the logic to handle togglebutton also.